oauth/authorization.md

```diff
@@ -1,92 +1,231 @@
-
 ### Projected Time
-
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
-
+About 1.5 hour
+- Lesson: 30 min
+- Guided Practice: 30 min	
+- Independent Practice: 15 min	
+- Check for Understanding: 15 min	
 ### Prerequisites
+Here are links to lessons that should be completed before this lesson:	
+- [Adding Auth0 to your Project](https://github.com/Techtonica/curriculum/blob/master/oauth/o-auth.md) - Especially the 'Authentication v. Authorization' slides	
+- [Node.js](/node-js/node-js.md)	
+- [Express.js](/express-js/express.md)	
 
-Here are links to lessons that should be completed before this lesson:
-
-- [Adding Auth0 to your Project](/oauth/o-auth.md) - Especially the 'Authentication v. Authorization' slides
-- [Node.js](/node-js/node-js.md)
-- [Express.js](/express-js/express.md)
 ### Motivation
-
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
-
-### Objectives
-
-**Participants will be able to:**
-
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
-
-### Specific Things To Teach
-
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
-
-### Materials
-
-- [Authorization Slides](https://docs.google.com/presentation/d/1p7fa20o7lRyvXuXhsFz8MfHktY5MaqSPSrLdNjocNcE/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
-
-### Lesson
-
+Many app developers need a way to persist user information from session to session, but they also want to provide a smooth experience for those users. The OAuth protocol allows user to "log in" to an app using their Google account (or other social media accounts). This removes friction for the user because they don't need to create a new profile or remember a new password. OAuth can be used for other types of integration, like allowing users to give an app permission to access the user's bank balances or Dropbox files. This is beyond the scope of this lesson but may be valuable to you in the future.	
+### Objectives	
+**Participants will be able to:**	
+- add authentication and authorization to an Express.js API.	
+### Specific Things To Teach	
+- Configure Auth0 APIs	
+- Validate Access Tokens
+- Protect API Endpoints	
+###  Materials
+- [Authorization Slides](https://docs.google.com/presentation/d/1p7fa20o7lRyvXuXhsFz8MfHktY5MaqSPSrLdNjocNcE/edit?usp=sharing)	
```

Comment:
I apologize for providing you the wrong link.  Please create a slideshow at https://docs.google.com/presentation/d/1mBXA2GqsARbR_LsouEEshNM5JvnQjDmrSXWhVeilfpc/edit?usp=sharing